src/pomodoro/Timer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import TimerDigits from './TimerDigits';
+import { useTimerReducer } from './useTimerReducer';
+import IncreaseTimeContainer from './IncreaseTimeContainer';
+import Button from './Button';
+import TimerControls from './TimerControls';
+
+export default function Timer() {
+  const {
+    time,
+    isActivated,
+    timerSchema,
+    start,
+    stop,
+    reset,
+    decreaseTime,
+    increaseTimeByOneMin,
+    increaseTimeByFiveMin,
+    increaseTimeByTwentyMin,
+  } = useTimerReducer();
+
+  useEffect(() => {
+    if (isActivated) {
+      const intervalID = setInterval(decreaseTime, 1000);
+      return () => clearInterval(intervalID);
+    }
+  }, [isActivated]);
+
+  return (
+    <div>
+      <TimerDigits time={time} />
+
+      <IncreaseTimeContainer>
+        <Button
+          label='1'
+          onClick={increaseTimeByOneMin}
+          className='Button Button--type-square'
+        />
+        <Button
+          label='5'
+          onClick={increaseTimeByFiveMin}
+          className='Button Button--type-square'
+        />
+        <Button
+          label='20'
+          onClick={increaseTimeByTwentyMin}
+          className='Button Button--type-square'
+        />
+      </IncreaseTimeContainer>
+
+      <TimerControls>
+        <Button
+          label='Change Time Schema'
```

Comment:
Weird word casing for a label

Reply:
I didn't know how to name this. Time Schema  means for example 25/5/15/4 Work Time, Break Time, Long Break Time and the number of short breaks before long break. So I think schema is quite good name